apps/src/code-studio/components/ShareDialog.jsx

```diff
@@ -27,6 +32,21 @@ const styles = {
     fontSize: 13,
     fontWeight: 'bold'
   },
+  button: {
+    backgroundColor: color.purple,
+    borderWidth: 0,
+    color: color.white,
+    fontSize: 'larger',
+    paddingTop: 12.5,
+    paddingBottom: 12.5,
```

Comment:
Are fractional pixels necessary? Would worry that different browsers are going to handle this differently.